common/static/common/templates/discussion/new-post.underscore

```diff
@@ -29,6 +29,12 @@
             </div>
         </label>
     </div>
+    <% } else if (is_cohorted) { %>
+    <div class="post-field group-visibility">
+        <label class="field-label">
+            <span class="field-label-text"></span>
+	</label>
+    </div>
```

Comment:
Its purpose is simply to act as a wrapper for the message, which is then filled out by a separate template.